ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -61,20 +61,20 @@ public struct ActivityIndicator: View, ConfigurableTokenizedControl {
         SemiRing(color: color,
                  thickness: tokens.thickness,
                  accessibilityLabel: accessibilityLabel)
-            .modifyIf(state.isAnimating, { animatedView in
+            .modifyIf(configuration.isAnimating, { animatedView in
                 animatedView
                     .rotationEffect(.degrees(rotationAngle), anchor: .center)
                     .onAppear {
                         startAnimation()
                     }
             })
-            .modifyIf(!state.isAnimating) { staticView in
+            .modifyIf(!configuration.isAnimating) { staticView in
```

Comment:
It seems like this is one case where configuration makes less sense as a name. Not sure if we should split variables that actually represent the state the control is in and variables that decided how the control are configured in to different classes, though.

Reply:
either a separate bug to go clean up but I do agree it seems weird that this is configuration rather than the actual state of the control like isHidden

Reply:
Yeah this definitely isn't 100% perfect. I wonder if properties like that would make more sense living on the SwiftUI control itself, rather than as part of a state/configuration object?